tests/integration/ovs_test.py

```diff
@@ -278,6 +314,9 @@ def test_create_and_remove_lag(self, port0_up, port1_up):
 
         with bridge.create() as state:
             assertlib.assert_state_match(state)
```

Comment:
It would be great to have similar tests for all other types like bond, OVS Bridge, Linux Bridge and Team - essentially all types that also have a name attribute.

Reply:
Bond, team does not have such needs yet.

Add test case for linux bridge vlan: `TestVlanFiltering::test_pretty_state_port_name_first`